db/migrate/20230309121510_create_verify_phones.rb

```diff
@@ -0,0 +1,9 @@
+class CreateVerifyPhones < ActiveRecord::Migration[7.0]
+    def change
+     create_table :verify_phones do |t|
+       t.string :number, null: false, unique: true, index: true, length: 10
```

Comment:
I debated over having this field as an integer or string but I settled on a string because it was easier to handle the model validations on a string that would be coming from the front end.